src/core/emotionInterface/emotionInterfaceModule.cpp

```diff
@@ -243,6 +245,35 @@ bool EmotionInterfaceModule::configure(ResourceFinder& config){
             _bitmask_emotion_table[id] = hexCode;
         }
 
+    }
+    
+	yarp::os::Property rcb_face_conf{ {"device",Value("remote_controlboard")},
+                                      {"local", Value(getName("/face/remoteControlBoard"))},
+                                      {"remote",Value("/icub/face")},
+                                      {"part",Value("face")} };
+
+    if (_poly.open(rcb_face_conf))
+    {
+        bool ok{ true };
+        int numAxes{ 0 };
+        yarp::dev::IControlMode* iCM{ nullptr };
+        yarp::dev::IControlLimits* iCtrlLim{ nullptr };
+        ok &= _poly.view(iCM);
+        ok &= _poly.view(_iPos);
+        ok &= _poly.view(iCtrlLim);
+        if (iCM)
+            ok &= iCM->setControlMode(_joint_eylids, VOCAB_CM_POSITION);
+        if (iCtrlLim)
+            ok &= iCtrlLim->getLimits(_joint_eylids, &_min, &_max);
+        if (!ok)
+        {
+            yError() << "Fail to configure correctly the remote_controlboard";
+            return false;
+        }
```

Comment:
Didn't we say that for RFE robot w/ motors off we should only raise a warning? This should cover the not infrequent conditions when at the test bench only LED's will need verification.

cc @marcoaccame

Reply:
If I did it correctly it should try to open the polidriver, and the fail of this open is **not** fatal.
It is **fatal** if once open it can't be configured correctly(view of the interfaces), that is very rare

Reply:
Ok then! I would do a specific test on this condition.